### ISSUE TITLE
Fix all stories currently present in the project

### ISSUE THREAD

**Issue by suyash-naithani** [streamshub/console#204]
After migrating all the stories into the project, most of them do not work due to various errors. Go through all the stories and fix them so they can be viewed using storybook

**Comment by MikeEdgar** [comment]
@riccardo-forina @hemahg is this still an issue?

**Comment by riccardo-forina** [comment]
Yeah, I've added something back over time but not all.

**Comment by hemahg** [comment]
yes, I added a few stories, but still few are left. I will work on it

**Comment by MikeEdgar** [comment]
@hemahg can we close this?

**Comment by hemahg** [comment]
yes, I have added story for all the components
